packages/web-components/fast-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../styles";
+
+export const RadioGroupStyles = css`
+    ${display("flex")} :host {
+        align-items: flex-start;
```

Comment:
Clarifying question - is flex doing anything on the host if we have a positioning region under it which is "flexed"? Do we need the positioning region? It seems like one of these is redundant. Consider that I want the radios to wrap at small viewports or whatnot, am I applying that behavior to the host element or to the part of "positioning region"?

Reply:
The host flex was to align the slotted label we wanted which could have a different alignment than the radios themselves. That was the thinking, if we didn't have the label slot it wouldn't be needed or only one flex layout would be needed.